### PR TITLE
Long range radio hear for observers.

### PR DESCRIPTION
**Pull request by eswordthecat**
An addition to #3638.

### CODE REVIEW

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@ var/global/list/player_list = list()				//List of all mobs **with clients attach
 var/global/list/mob_list = list()					//List of all mobs, including clientless
 var/global/list/living_mob_list = list()			//List of all alive mobs, including clientless. Excludes /mob/new_player
 var/global/list/dead_mob_list = list()				//List of all dead mobs, including clientless. Excludes /mob/new_player
+var/list/observers = new/list()
```
Comment (ComicIronic) [inline code/__HELPERS/global_lists.dm]: Why not just use dead_mob_list?

Comment (eswordthecat) [inline code/__HELPERS/global_lists.dm]: >>//List of all dead mobs, including clientless.
Well...
>>Excludes /mob/new_player
Is a lie.

Comment (eswordthecat) [inline code/__HELPERS/global_lists.dm]: Also I think observer must not be treaten as dead.

**Comment by eswordthecat** [comment]
Can be pulled.

**Pull request merged**